app/controllers/admin/facility_groups_controller.rb

```diff
@@ -3,10 +3,6 @@ class Admin::FacilityGroupsController < AdminController
   before_action :set_organizations, only: [:new, :edit, :update, :create]
   before_action :set_protocols, only: [:new, :edit, :update, :create]
 
-  skip_after_action :verify_authorized
-  skip_after_action :verify_policy_scoped
-  after_action :verify_authorization_attempted
-
```

Comment:
These ^ are no longer necessary at a per-controller level, since they're handled by the AdminController